### DISCUSSION TITLE
Spreadsheet "Find" Feature

### DISCUSSION THREAD

**Discussion by Gpwaran** [standardnotes/forum#3112] [General]
Hi,

I cant seem to find the "Find" feature in spreadsheets - while using PC or Android.

Ive tried to look through all drop down menus. Seems like it would be a common feature for a Spreadsheet.

Apologize if this question has been answered before

**Comment by effieeee** [comment]
Hi @Gpwaran FIND and SEARCH are both available functions on the Spreadsheets so far, though were you perhaps referring to a different function? Please also note that Spreadsheets does not have an in-note search function 🙏

**Comment by Gpwaran** [comment]
Sorry,

I guess I'm looking for the "CTRL+F" equivalent. Say I have many cells and I want to find a specific cell

**Reply by effieeee** [reply]
I see, in that case though, I'm afraid that the Spreadsheets really don't have this capability at the moment :(
